### DISCUSSION TITLE
Not connected errors in PET files

### DISCUSSION THREAD

**Discussion by ericaligo-NOAA** [ufs-community/ufs-weather-model#1709] [Q&A]
I'm getting errors in the PET files that I've never seen before. Any ideas what this suggests? Here's a sample of those errors:
20230412 170601.379 INFO             PET000 ... returned from wrtFB(03,01) FieldBundleRegridStore().
20230412 170601.470 INFO             PET000 Driver is in ModifyCplLists()
20230412 170601.470 INFO             PET000  Found            0 Connectors. Modifying CplList Attribute....
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_pres_interface is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_pres_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_geop_interface is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_geop_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_temp_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_zonal_wind_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_merid_wind_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_omega_levels is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_tracer_mass_frac is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field soil_type is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_pbl_height is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field surface_cell_area is not connected
20230412 170601.470 INFO             PET000 module_cplfields.F90:527 realizeConnectedCplFields FV3 Export Field inst_convective_rainfall_amount is not connected

**Comment by DeniseWorthen** [comment]
Those are informational messages, not errors:

https://github.com/NOAA-EMC/fv3atm/blob/f964b4b5e4807363f891c8b2e7a10a8534f109a6/cpl/module_cplfields.F90#L523-L527

**Comment by ericaligo-NOAA** [comment]
Okay. Thanks.  I've never seen those messages in the PET files. Any 
ideas what it might mean? It points to line 527, but I have no idea why 
it says it's 'not connected'. In other runs that don't print this 
message, there is a list of fields and nothing about 'not connected'

line 527 of FV3/cpl/module_cplfields.F90. The code is here: 
/scratch2/NCEPDEV/fv3-cam/save/Eric.Aligo/FA_03_28_2023
483             call ESMF_FieldEmptyComplete(field, 
typekind=ESMF_TYPEKIND_R8, &
484                                      ungriddedLBound=(/1/), 
ungriddedUBound=(/numLevels+1/), rc=rc)
485             if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
486           case ('t','tracer')
487             call ESMF_FieldEmptyComplete(field, 
typekind=ESMF_TYPEKIND_R8, &
488                                      ungriddedLBound=(/1, 1/), 
ungriddedUBound=(/numLevels, numTracers/), rc=rc)
489             if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
490             if (allocated(tracerNames)) then
491               call addFieldMetadata(field, 'tracerNames', 
tracerNames, rc=rc)
492               if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
493             end if
494             if (allocated(tracerUnits)) then
495               call addFieldMetadata(field, 'tracerUnits', 
tracerUnits, rc=rc)
496               if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
497             end if
498           case ('s','surface')
499             call ESMF_FieldEmptyComplete(field, 
typekind=ESMF_TYPEKIND_R8, rc=rc)
500             if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
501           case ('g','soil')
502             call ESMF_FieldEmptyComplete(field, 
typekind=ESMF_TYPEKIND_R8, &
503                                      ungriddedLBound=(/1/), 
ungriddedUBound=(/numSoilLayers/), rc=rc)
504             if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
505           case default
506             call ESMF_LogSetError(ESMF_RC_NOT_VALID, &
507               msg="exportFieldType = 
'"//trim(fields_info(item)%type)//"' not recognized", &
508               line=__LINE__, file=__FILE__, rcToReturn=rc)
509             return
510         end select
511         call NUOPC_Realize(state, field=field, rc=rc)
512         if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
513
514         ! -- initialize field value
515         call ESMF_FieldFill(field, dataFillScheme="const", 
const1=l_fill_value, rc=rc)
516         if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
517
518         ! -- save field
519         fieldList(item) = field
520         call ESMF_LogWrite('realizeConnectedCplFields 
'//trim(state_tag)//' Field '//trim(fields_info(item)%name) &
521                            // ' is connected ', ESMF_LOGMSG_INFO, 
line=__LINE__, file=__FILE__, rc=rc)
522       else
523         ! remove a not connected Field from State
524         call ESMF_StateRemove(state, 
(/trim(fields_info(item)%name)/), rc=rc)
525         if (ESMF_LogFoundError(rcToCheck=rc, 
msg=ESMF_LOGERR_PASSTHRU, line=__LINE__, file=__FILE__)) return
526         call ESMF_LogWrite('realizeConnectedCplFields 
'//trim(state_tag)//' Field '//trim(fields_info(item)%name) &
*527                            // ' is not connected ', 
ESMF_LOGMSG_INFO, line=__LINE__, file=__FILE__, rc=rc)*
528       end if
529     end do

**Comment by DeniseWorthen** [comment]
Are you running a regression test? Which one?
